content/chapitres/ci.adoc

```diff
@@ -164,25 +219,25 @@ include::../code-samples/gh-actions/say-hello-full.yml[tags="common,checkout,sho
 
 == Exemple : Environnement d'exécution
 
-* Notre pipeline de build dit que "la vache" doit afficher le contenu du fichier `README.md`
-** WAT (Non, nous ne sommes pas fous) ?
+* Notre pipeline de build doit s'assurer que "la vache" 🐮 doit nous lire 💬 le contenu du fichier `README.md`
+** link:https://www.destroyallsoftware.com/talks/wat[WAT 🤪, window="_blank"] ?
```

Comment:
YES. THANK YOU FOR THIS.

Reply:
Though we should be careful: not everyone might understand english ?

Reply:
It's an easter egg, so no worry :)